### PR TITLE
Allow optional fs-extra#mkdirs options object

### PR DESCRIPTION
**Pull request by caitp**

### CODE REVIEW

File: fs-extra/fs-extra.d.ts [hunk 1]
```diff
@@ -171,6 +173,11 @@ declare module "fs-extra" {
 		flag?: string;
 	}
 
+	export interface MkdirOptions {
+		fs?: any;
```
Comment (caitp) [inline fs-extra/fs-extra.d.ts]: This is important for some unit tests which rely on fs-extra and mock-fs. So a merge would be super helpful :)

**Comment by vvakame** [comment]
*fs-extra/fs-extra.d.ts*

to author(@midknight41). could you review this PR?
:+1: or :-1:?

check list

* [ ] pass the Travic-CI test?

**Comment by caitp** [comment]
>  pass the Travic-CI test?

Check?

**Comment by midknight41** [comment]
LGTM

**Comment by vvakame** [comment]
@midknight41 thanks!
@caitp thanks mate! :+1:

**Pull request merged**